src/cluster_info.rs

```diff
@@ -274,6 +296,17 @@ impl ClusterInfo {
             .collect()
     }
 
+    pub fn sorted_tvu_peers(&self, bank: &Arc<Bank>) -> Vec<NodeInfo> {
+        let peers = self.tvu_peers();
+        let mut peers_with_stakes: Vec<(&NodeInfo, u64)> =
+            peers.iter().map(|c| (c, bank.get_stake(&c.id))).collect();
+        peers_with_stakes.sort_unstable_by(|l, r| r.1.cmp(&l.1));
```

Comment:
if stakes are equal, you can compare ids in order to make this stable (we'd want the same answer on all boxes, right?)